app/code/Magento/MediaGalleryRenditions/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="system">
+            <group id="media_gallery_renditions" translate="label" type="text" sortOrder="1010" showInDefault="1" showInWebsite="0" showInStore="0">
+                <label>Media Gallery Renditions</label>
```

Comment:
I believe it would be nice to add some description of the functionality.

Reply:
@belbiy is preparing the description, it will be added in a separate PR